sunpy/io/tests/test_ana.py

```diff
@@ -64,18 +60,18 @@ def test_f32u():
     afilename = tempfile.NamedTemporaryFile().name
     ana.write(afilename, img_f32, 'testcase', 0)
     img_f32u_rec = ana.read(afilename)
-    assert np.sum(img_f32u_rec[0][0]- img_f32) == 0
+    assert np.sum(img_f32u_rec[0][0] - img_f32) == 0
 
 
-@skip_ana
-def test_f32c():
-    # Test if float 32 compressed functions
-    #TODO: Bug with same code. Needs to be tracked down.
-
+# TODO: Bug with same code. Needs to be tracked down.
```

Comment:
Is this why the coverage has dropped? Why is this breaking?

Reply:
Yeah, this is probably why the coverage is down. I'm not sure why, clearly something I didn't fix when we merged in the code, way back when.